linker/shared/src/main/scala/org/scalajs/linker/frontend/optimizer/OptimizerCore.scala

```diff
@@ -1962,8 +1961,9 @@ private[optimizer] abstract class OptimizerCore(config: CommonPhaseConfig) {
     } (cont) (scope.withEnv(OptEnv.Empty))
   }
 
-  private def callIntrinsic(code: Int, optTReceiver: Option[PreTransform],
-      targs: List[PreTransform], isStat: Boolean, usePreTransform: Boolean)(
+  private def callIntrinsic(code: Int, flags: ApplyFlags,
+      optTReceiver: Option[PreTransform], targs: List[PreTransform],
+      isStat: Boolean, usePreTransform: Boolean)(
```

Comment:
It seems like the call-sites of this should guard against bad apply flags, but currently don't.